modules/ROOT/pages/database-administration/standard-databases/alter-databases.adoc

```diff
@@ -97,11 +131,42 @@ ALTER DATABASE nonExisting IF EXISTS
 SET ACCESS READ WRITE
 ----
 
+[role=label--new-2025.06]
+[[alter-database-default-language]]
+== Alter database default language
+
+You can change the default Cypher version of an existing database, including the `system` database, using the `ALTER DATABASE` command with the `SET DEFAULT LANGUAGE` clause.
+For example:
+
+[source, cypher]
+----
+ALTER DATABASE movies SET DEFAULT LANGUAGE CYPHER 25
+----
+This command will change the default Cypher version of the database `movies` to Cypher 25.
+
+include::partial$/view-cypher-version.adoc[]
+
+.Result
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| name | defaultLanguage
+
+| "actors"  | "CYPHER 25"
+| "movies"  | "CYPHER 25"
+| "neo4j"   | "CYPHER 25"
+| "system"  | "CYPHER 25"
+
+2+d|Rows: 4
+|===
+
+
+[role=label--enterprise-edition label--not-on-aura]
 == Alter database topology
 
 In a cluster environment, you can use the `ALTER DATABASE` command to change the number of servers hosting a database.
 For more information, see xref::clustering/databases.adoc#alter-topology[Managing databases in a cluster].
 
+[role=label--enterprise-edition label--not-on-aura]
 [[alter-database-options]]
 == `ALTER DATABASE` options
```

Comment:
```suggestion
== Alter database options
```
not really relevant for this PR, just noticed it was inconsistent with the other headers:
* Alter database topology
* Alter database default language
* Alter database access mode
vs
* ALTER DATABASE options